### ISSUE TITLE
Add achievements and/or badges

### ISSUE THREAD

**Issue by davidfowl** [JabbR/JabbR#134]
People go crazy when they get achievemens. We need to add something like this to get people coming back :).

**Comment by kiliman** [comment]
Yeah, need a "Lurker" achievement... longest time in room without posting a message :)

**Comment by Buildstarted** [comment]
Heh, just about everyone will have that badge :)

**Comment by mikeminutillo** [comment]
Remember that badges should reinforce good behavior not bad so "Last One Out" or "Lurker" aren't great.

They call me Mr. Tibbs: Set your nick
Here's lookin' at you: Set your gravatar
The Host: Start a room with 5 active participants

**Comment by jschwarty** [comment]
"Multiple Personalities" - You have changed your /nick more than 5 times
"Sobering Up" - Managed to go 30 days without embedding a YouTube video
"Let's Take This Outside" - Made your first private room

**Comment by jongalloway** [comment]
Promoter badge of some kind - if you can detect if people tweet a link to a room. 

Also - would be easier to track if you had a trackable social sharing link to a room. http://jabbr.net/#/rooms/general-chat?ref=jongalloway or something.
